components/crowdfunding-redesign/edit/common.tsx

```diff
@@ -0,0 +1,236 @@
+    editAccountSetting(account: $account, key: "crowdfundingRedesign", value: $value) {
+      id
+      settings
+    }
+  }
+`;
+
+export const MainDetailsForm = ({ initialValues, schema, onSubmit }) => {
+  return (
+    <FormikZod
+      schema={schema.pick({ name: true, description: true, primaryColor: true })}
+      initialValues={initialValues}
+      onSubmit={onSubmit}
+    >
+      {(formik: FormikProps<z.infer<typeof schema>>) => {
+        return (
+          <Form>
+            <div className="flex flex-col items-start gap-4">
+              <FormField name="name" label="Title" />
+              <FormField
+                name="description"
+                label="Subtitle"
+                hint="Large, bold text that appears at the top of your page."
+              />
+              <FormField name="primaryColor" label="Theme color">
+                {({ field }) => {
+                  return <ColorPicker {...field} onChange={color => formik.setFieldValue('primaryColor', color)} />;
+                }}
+              </FormField>
+              <Button type="submit" loading={formik.isSubmitting}>
+                <FormattedMessage defaultMessage="Save" id="save" />
+              </Button>
+            </div>
+          </Form>
+        );
+      }}
+    </FormikZod>
+  );
+};
+
+export const LongDescriptionForm = ({ initialValues, schema, onSubmit }) => {
+  return (
+    <FormikZod schema={schema.pick({ longDescription: true })} initialValues={initialValues} onSubmit={onSubmit}>
+      {(formik: FormikProps<z.infer<typeof schema>>) => (
+        <Form>
+          <div className="flex flex-col gap-4">
+            <div className="flex justify-between gap-4">
+              <div>
+                <div className="text-lg font-medium">
+                  <FormattedMessage defaultMessage="About" id="collective.about.title" />
+                </div>
+                <p className="text-sm text-muted-foreground">
+                  <FormattedMessage defaultMessage="Tell your story and explain your purpose." id="SectionAbout.Why" />
+                </p>
+              </div>
+              <Button type="submit" loading={formik.isSubmitting}>
+                <FormattedMessage defaultMessage="Save" id="save" />
+              </Button>
+            </div>
+            <FormField name="longDescription">
+              {({ field }) => {
+                return (
+                  <RichTextEditor
+                    kind="ACCOUNT_LONG_DESCRIPTION"
+                    {...field}
+                    defaultValue={field.value}
+                    onChange={e => formik.setFieldValue('longDescription', e.target.value)}
+                    placeholder="Tell your story..."
+                    toolbarBackgroundColor="#F7F8FA"
+                    withStickyToolbar
+                    videoEmbedEnabled
+                    autoFocus
+                    toolbarOffsetY={0}
+                    withBorders
+                  />
```

Comment:
The sticky behavior for the rich text is not working well, because a parent with `position=relative` prevents it from being at the top of the page.

https://github.com/opencollective/opencollective-frontend/assets/1556356/57a3e690-6dbf-411c-aa44-a8a6644c2360

Also, I don't think autofocus is suited here since this is not the 1st field in the form.

As a simplification for now, I'd suggest:

```suggestion
                  <RichTextEditor
                    kind="ACCOUNT_LONG_DESCRIPTION"
                    {...field}
                    defaultValue={field.value}
                    onChange={e => formik.setFieldValue('longDescription', e.target.value)}
                    placeholder="Tell your story..."
                    toolbarBackgroundColor="#F7F8FA"
                    videoEmbedEnabled
                    withBorders
                  />
```

Reply:
Fixed this